README.md

```diff
@@ -1,24 +1,155 @@
+<ul>
+  <li><a href="https://html.com/">REACTJS</a></li>
+</ul>
+</details>
+<details><summary>Backend</summary>
+<ul>
+  <li><a href='https://rubyonrails.org/'>Rails</a></li>
+</ul>
+</details>
+<details><summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+### Key Features
 
-* Services (job queues, cache servers, search engines, etc.)
+-Display the random greeting with a react component.
 
-* Deployment instructions
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* ...
+## :computer: Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.<br>
+
+### Prerequisites
+
+You need the following tools be installed in your computer:
+
+- [Rails](https://guides.rubyonrails.org/)
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+- [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+- IDE
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+
+```sh
+  cd my-folder
+
+  git clone https://github.com/tomdan-ai/hello_rails_react.git
+```
+
+### Install
+
+Install this project with:<br>
+
+```sh
+  cd hello_rails_react
+
+  bundle install
+```
+
+### Database
+
+Create the databases and run migrations with:
+
+```sh
+rails db:create
+rails db:migrate
+rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  yarn build
+  rails server or rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :busts_in_silhouette: Authors <a name="authors"></a>
+
+:bust_in_silhouette: **Tom Daniel**
+
+- GitHub: [@githubhandle](https://github.com/tomdan-ai)
+- Twitter: [@twitterhandle](https://twitter.com/tomudoh1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tomudoh/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :telescope: Future Features <a name="future-features"></a>
+
+- Improve app UI
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :handshake: Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/tomdan-ai/hello_rails_react/issues).
+
+<!-- Feel free to check the [issues page](../../issues/). -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :star: Show your support <a name="support"></a>
+
+If you like this project give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :pray: Acknowledgements <a name="acknowledgements"></a>
+
+Thanks to Microverse for their support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+  
+## :memo: License <a name="license"></a>
+This project is [MIT](../MIT) licensed.
```

Comment:
- [ ] the link to the MIT license is not working as expected please fix it before requesting another review.